instance-service/src/main/java/space/swordfish/instance/service/service/EC2UserClientImpl.java

```diff
@@ -14,45 +15,43 @@
 @Service
 public class EC2UserClientImpl extends EC2BaseService implements EC2UserClient {
 
-    @Autowired
-    private AuthenticationService authenticationService;
+  @Autowired private AuthenticationService authenticationService;
 
-    @Override
-    public AmazonEC2Async amazonEC2Async() {
-        User user = authenticationService.getCurrentUser();
+  @Override
+  public AmazonEC2Async amazonEC2Async() {
+    User user = authenticationService.getCurrentUser();
 
-        String accessKey = user.getAwsKey();
-        String secretKey = user.getAwsSecret();
-        String region = user.getAwsRegion();
+    String accessKey = user.getAwsKey();
+    String secretKey = user.getAwsSecret();
+    String region = user.getAwsRegion();
 
-        if (secretKey.equals("") || accessKey.equals("") || region.equals("")) {
-            return null;
-        }
-
-        BasicAWSCredentials basicAWSCredentials = new BasicAWSCredentials(
-                accessKey, secretKey);
-
-        return AmazonEC2AsyncClientBuilder.standard().withRegion(region)
-                .withCredentials(new AWSStaticCredentialsProvider(basicAWSCredentials))
-                .build();
+    if (secretKey.equals("") || accessKey.equals("") || region.equals("")) {
+      return null;
     }
 
-    @Override
-    public AmazonEC2Async amazonEC2Async(User user) {
-        String accessKey = user.getAwsKey();
-        String secretKey = user.getAwsSecret();
-        String region = user.getAwsRegion();
+    BasicAWSCredentials basicAWSCredentials = new BasicAWSCredentials(accessKey, secretKey);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

If possible, use temporary security credentials (IAM roles) instead of long-term access keys.
Long-term access keys, such as those associated with IAM users and AWS account root users, remain valid until you manually revoke them. However, temporary security credentials obtained through IAM roles and other features of the AWS Security Token Service expire after a short period of time. Use temporary security credentials to help reduce your risk in case credentials are accidentally exposed.
Learn more about [best practices for managing AWS access keys](https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html).